contracts/0.8.25/vaults/interfaces/IStakingVault.sol

```diff
@@ -20,9 +20,17 @@ interface IStakingVault {
         int128 inOutDelta;
     }
 
+    struct Deposit {
+        bytes pubkey;
+        bytes signature;
+        uint256 amount;
+        bytes32 depositDataRoot;
```

Comment:
Since depositDataRoot is now not calculated, maybe need some info from where to take it? 
Will it be taken from the deposit_data file?

Reply:
It's probably even worth to include a view function that calculates the deposit_data_root to be conveniently used by external parties

Reply:
It would requrie a ton of code (memory lib, helpers like little-endian conversion, root computation itself) to be added to this contract just to be wrapped in an external function.